bot.py

```diff
@@ -112,6 +112,64 @@ async def list(ctx):
     )
 
 
+@bot.command()
+async def add(ctx):
+    author = ctx.message.author.id
+    await ctx.message.channel.send(
+        embed = Embed.from_dict(
+            {
+                "fields": [
+                    {
+                        "name": "Enter items to add to personal inventory:",
+                        "value": "\u200b"
+                    }
+                ]
```

Comment:
```suggestion
                "fields": [
                    {
                        "name": "Enter items to add to personal inventory:",
                        "value": "\u200b",
                    }
                ]
```

final items in multi-line lists should trail a `,` so when you add more items later, the diff of changes is less complex.